README.md

```diff
@@ -110,7 +112,19 @@ run the above commands with an additional option: `-d`, such as
 You could also specify the data file name with option `-f`, such as
 
     python3 demo_hypersearchDL.py -d /path-to-data-dir -f data.pkl
-    
+## Sample execution sequence of the bootstrapped internal validation for the deep learning model ([demo_validateDL.py](https://github.com/UK-Digital-Heart-Project/4Dsurvival/blob/master/demo/scripts/demo_validateDL.py)) using the container with latest lifelines package:
+`screen`\
+`docker pull lisurui6/4dsurvival-gpu:1.1`\
+Mount a drive to use data from outside of the containder:
+Here the folder '/mnt/.../sjadhav/.../data' will be visible inside the container:\
+`nvidia-docker run -ti --rm -v /mnt/.../sjadhav/.../data:/data lisurui6/4dsurvival-gpu:1.1`\
+`cd 4Dsurvival/demo`\
+Can edit a file (e.g demo_validateDL.py) and make changes (e.g.: changing the dropout range to [.1,.5] in step 1a and 2a of the code or reducing the number of bootstrap   sampling) using the editor vim:\
+`apt-get install vim`\
+`vim demo_validateDL.py`\
+ Execute the script with the available GPU card:\
+`CUDA_VISIBLE_DEVICES=2 python3 demo_validateDL.py`
+
```

Comment:
This has already been covered in the readme, it also contains directory that specific to you. Does not belong in the general readme.